types/react-relay/index.d.ts

```diff
@@ -11,20 +11,28 @@ export {
     commitLocalUpdate,
     commitRelayModernMutation as commitMutation,
     fetchRelayModernQuery as fetchQuery,
+    GraphQLTaggedNode,
     requestRelaySubscription as requestSubscription,
 } from "relay-runtime";
 
 import * as React from "react";
 import * as RelayRuntimeTypes from "relay-runtime";
 
+// ~~~~~~~~~~~~~~~~~~~~~
+// Utility types
+// ~~~~~~~~~~~~~~~~~~~~~
+
+// Taken from https://github.com/pelotom/type-zoo
```

Comment:
@kastermester The utility types of the same names that you used where removing the optionality modifiers from props, so went with these instead.